dataframe/dataframe_test.go

```diff
@@ -2558,3 +2559,76 @@ func TestDescribe(t *testing.T) {
 		}
 	}
 }
+
+func TestDataFrame_Insert(t *testing.T) {
+	tests := []struct {
+		desc     string
+		df       DataFrame
+		value    DataFrame
+		pos      int
+		expected string
+	}{
+		{
+			"TestDataFrame_Insert:0: DataframeString.Insert(DataframeString) & pos=end of Series",
+			LoadRecords(
+				[][]string{
+					{"A", "C", "D"},
+					{"1", "5.1", "true"},
+					{"NaN", "6.0", "true"},
+					{"2", "6.0", "false"},
+				},
+			),
+			LoadRecords(
+				[][]string{
+					{"A", "C", "D"},
+					{"2", "7.1", "false"},
+				},
+			),
+			-1,
+			`[4x3] DataFrame
+
+    A     C        D     
+ 0: 1     5.100000 true  
+ 1: NaN   6.000000 true  
+ 2: 2     6.000000 false 
+ 3: 2     7.100000 false 
+    <int> <float>  <bool>
+`,
```

Comment:
Setting the expectations like this is a bit dangerous, as changes in the print function will break it. Can you look at the rest of the testing code for a more consistent usage of expectations?

Reply:
have implemented the assertion using reflect.DeepEqual now